lib/Controller/Display.php

```diff
@@ -178,14 +185,86 @@ function displayPage(Request $request, Response $response)
 
         // Call to render the template
         $this->getState()->template = 'display-page';
+
+        $mapConfig = [
+        'setArea' => [
+            'lat' => 51.505,
```

Comment:
The CMS has a default lat/long which is available in global settings, we could use that here instead.